### PR TITLE
Add the ascii challenge, remove 2fa challenge, fix fuel challenge 

### PR DESCRIPTION
**Pull request by KaiDevrim**

### CODE REVIEW

File: src/challenges/oxygen.ts [hunk 1]
```diff
@@ -41,7 +40,7 @@ export default {
 
 Thankfully, the ship's engineers built in a backup oxygen reserve, but it's rather difficult to access.
 
-To switch your ship's oxygen over to the backup, you'll need to make a POST request to a URL that's hidden in this message.`,
+To switch your ship's oxygen over to the backup, you'll need to make a POST request to this spooky url https://epoch-mission-control.herokuapp.com/oxygen/6${ctx.team.id}763.`,
```
Comment (sampoder) [inline src/challenges/oxygen.ts]:
```suggestion
To switch your ship's oxygen over to the backup, you'll need to make a POST request to this spooky URL: https://epoch-mission-control.herokuapp.com/oxygen/6${ctx.team.id}763.`,
```

File: src/challenges/random.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+        ctx.listener.event("message", onMessage);
+
+        return () => {
+            ctx.listener.removeListener("event:message", onMessage);
+        };
+    },
+    async start(ctx: ChallengeContext) {
+        await ctx.post(`0.5390815646058106, 0.2891964436397205, 0.030... It seems like your computer broke right as you were about to generate the last few digits of the initialization sequence. Thankfully you set the seed to the current date. Now what was
+Enter the last few digits of the third random number down in the chat :arrow_down: (P.S), make sure to use python to generate the random numbers.`);
```
Comment (sampoder) [inline src/challenges/random.ts]:
```suggestion
Enter the last few digits of the third random number down in the chat :arrow_down: (PS: make sure to use Python to generate the random numbers)`);
```

File: src/challenges/random.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        };
+
+        ctx.listener.event("message", onMessage);
+
+        return () => {
+            ctx.listener.removeListener("event:message", onMessage);
+        };
+    },
+    async start(ctx: ChallengeContext) {
+        await ctx.post(`0.5390815646058106, 0.2891964436397205, 0.030... It seems like your computer broke right as you were about to generate the last few digits of the initialization sequence. Thankfully you set the seed to the current date. Now what was
```
Comment (sampoder) [inline src/challenges/random.ts]:
```suggestion
        await ctx.post(`0.5390815646058106, 0.2891964436397205, 0.030... It seems like your computer broke right as you were about to generate the last few digits of the initialization sequence. Thankfully you set the seed to the current date.
```
FYI: in Delhi, it'll be the 31st

**Review by sampoder — CHANGES_REQUESTED** [review]
@KaiDevrim a couple of suggestions & comments from me. Could you also document how to solve the challenges in a comment in this PR?

**Comment by KaiDevrim** [comment]
@sampoder the random one is simple, just do
```py
import random
random.seed(30)
print(random.random())
```

the ascii one is just as simple as it looks, just translate the numbers it gives you into letters.

**Pull request merged**